JeMPI_Apps/JeMPI_UI/src/components/dashboard/widgets/MandUWidget.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box } from '@mui/material'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Paper from '@mui/material/Paper'
+
+function createData(fieldName: string, mValue: number, uValue: number) {
+  return { fieldName, mValue, uValue }
+}
+const rows = [
+  createData('Given Name', 75, 6.0),
+  createData('Family Name', 237, 9.0),
+  createData('Gender', 0.5, 0.7),
+  createData('Birthdate', 305, 3.7),
+  createData('Phone', 305, 3.7),
+  createData('National Id ', 305, 3.7),
+  createData('City ', 305, 3.7)
+]
+function MandU() {
+  return (
+    <Box component={'fieldset'}>
+      <legend>M & U Values</legend>
+      <TableContainer component={Paper} sx={{ minHeight: '450px' }}>
+        <Table aria-label="simple table">
```

Comment:
I'm pressuming this table is responsive/can accomodate different screen sizes?

Reply:
@walisc Yes, tested on all devices (XS, SM, MD, LG, XL, XXL ), mostly minimum height causes no issue with responsive web.